src/style.css

```diff
@@ -1,3 +1,90 @@
+html {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+
+  --main-color: #00acdc;
+
+  text-decoration: none;
+  font-family: Roboto, sans-serif;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: 500;
+}
+
 body {
   margin: 0;
 }
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  padding: 0 50px;
+}
+
+.logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+  padding: 10px 0;
+}
+
+.navigation {
+  display: block;
+}
+
+.nav__list {
+  margin: 0;
+  display: flex;
+  list-style: none;
+  padding: 0;
+}
+
+.nav__item {
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin: 0;
+}
+
+.nav__item a {
+  padding: 23px 0;
+}
```

Comment:
Fix it
```suggestion
```
<img width="533" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/64750958/74ff7c14-0140-4eae-b7a9-2594b1e0252d">